src/inspector_io.h

```diff
@@ -48,7 +48,8 @@ class InspectorIo {
   static std::unique_ptr<InspectorIo> Start(
       std::shared_ptr<MainThreadHandle> main_thread,
       const std::string& path,
-      std::shared_ptr<HostPort> host_port);
+      std::shared_ptr<HostPort> host_port,
+      const std::string& inspect_publish_uid);
```

Comment:
I believe that this class should not be parsing CLI parameter values but take two booleans instead. I would suggest parsing them as close to the node_options.cc as possible.

Reply:
I moved parsing to node_options.cc, since we need to parse it there in any case to validate argument value.